daemon/graphdriver/devmapper/deviceset.go

```diff
@@ -1700,6 +1706,23 @@ func (devices *DeviceSet) Status() *Status {
 	return status
 }
 
+// Status returns the current status of this deviceset
+func (devices *DeviceSet) ExportDeviceMetadata(hash string) (*DeviceMetadata, error) {
+	info, err := devices.lookupDevice(hash)
+	if err != nil {
+		return nil, err
+	}
+
+	info.lock.Lock()
```

Comment:
Is this lock necessary? Is the info ever modified concurrently after loading it?